Trackers/centroid_tracker/centroid.py

```diff
@@ -113,4 +102,35 @@ def update(self, detections):
                 for col in unusedCols:
                     self.add_track(self.nextID, detections[col], inputCentroids[col])
         self.delete_track()
-        return self.tracks
+        result=[a for a in self.tracks if a.hits>=self.min_hits]
+        return result
+
+    def find_match(self,objectCentroid,inputCentroids):
+        """
+        Returns list of matched objects , unmatched tracks and unmatched detections.  
+        
+        Args
+        objectCentroid : list of centroids of existing objects.
+        inputCentroids : list of centroids of current detections.
+
+        Returns
+        unusedcols : index of newly detected tracks
+        matched : list of index of matching objects 
+        """
+        D = dist.cdist(np.array(objectCentroid), inputCentroids)
+        rows = D.min(axis=1).argsort()
+        cols = D.argmin(axis=1)[rows]
+        usedRows = set()
+        usedCols = set()
+        matched =[]
```

Comment:
use variables such as `matches`, `unmatched_tracks` , `unmatched_detections`  to have a uniform naming convention.